$NetBSD$

- Fix library name of jbig2dec (no 'lib' prefix is needed)
- Adjust library name of mupdf-third for mupdf>=1.14.0.

--- meson.build.orig	2018-09-29 16:16:56.000000000 +0000
+++ meson.build
@@ -20,7 +20,7 @@ girara = dependency('girara-gtk3')
 glib = dependency('glib-2.0')
 cairo = dependency('cairo')
 mupdf = cc.find_library('mupdf')
-mupdfthird = cc.find_library('mupdfthird')
+mupdfthird = cc.find_library('mupdf-third')
 
 build_dependencies = [
   zathura,
@@ -33,7 +33,7 @@ build_dependencies = [
 
 if get_option('link-external')
   libjpeg = dependency('libjpeg')
-  libjbig2dec = cc.find_library('libjbig2dec')
+  libjbig2dec = cc.find_library('jbig2dec')
   libopenjp2 = dependency('libopenjp2')
 
   build_dependencies += [
